tools/lib/test_script.py

```diff
@@ -122,9 +120,6 @@ def find_js_test_files(test_dir: str, files: Iterable[str]) -> List[str]:
 
 def prepare_puppeteer_run(is_firefox: bool = False) -> None:
     os.chdir(ZULIP_PATH)
-    download_path = platformdirs.user_cache_dir("zulip/puppeteer-download")
-    os.makedirs(download_path, exist_ok=True)
-    os.environ["PUPPETEER_DOWNLOAD_PATH"] = download_path
```

Comment:
Why is this no longer required?

Reply:
It was upstreamed in Puppeteer 19.0.0 (puppeteer/puppeteer#9095).

Reply:
I see; the path changes but in a way that should be totally fine.